BUILD.gn

```diff
@@ -8,16 +8,33 @@ challenge_bypass_build_path = "$root_gen_dir/challenge_bypass_ristretto/out"
 
 challenge_bypass_lib_path = challenge_bypass_build_path
 
+challenge_bypass_platform = ""
+
 if (is_win) {
   challenge_bypass_lib_path += "/x86_64-pc-windows-msvc"
+  challenge_bypass_platform = "Windows"
 } else if (is_mac) {
   challenge_bypass_lib_path += "/x86_64-apple-darwin"
+  challenge_bypass_platform = "macOS"
 } else if (is_linux) {
-  challenge_bypass_lib_path += "/x86_64-unknown-linux-musl"
+  challenge_bypass_lib_path += "/x86_64-unknown-linux-gnu"
+  challenge_bypass_platform = "Linux"
 } else if (is_android) {
-  challenge_bypass_lib_path += "/x86_64-linux-android"
+  if (target_cpu == "x86") {
+    challenge_bypass_lib_path += "/x86_64-linux-android"
+    challenge_bypass_platform = "Android x86_64"
+  } else {
+    challenge_bypass_lib_path += "/arm-linux-androideabi"
+    challenge_bypass_platform = "Android"
+  }
 } else if (is_ios) {
-  challenge_bypass_lib_path += "/x86_64-apple-ios"
+  if (target_cpu != "x64") {
```

Comment:
I think it's arm/arm64 for both ios and and android

Reply:
if x64 we build for simulator and if not x64 use the latest aarch64-apple-ios target. Brave Browser is iOS 12 and above so we can safely use this target

Reply:
Android we do use arm-linux-androideabi unless the target is x86